app/uk/gov/hmrc/trackingconsentfrontend/assets/src/loaders/message-format-loader/index.js

```diff
@@ -0,0 +1,21 @@
+const {parse} = require('dot-properties')
+
+const stripDoubleQuotesFromString = content => content.replace(/''/, "'")
```

Comment:
Typographic quotes should be used in services which would mean this isn't necessary.

Reply:
This isn't related to that, it's to properly translate escaped single quotes in Scala message format files.

Reply:
Yeah but I don't think there should be any normal straighty quotes in your messages file, only curly/typographic quotes.

Reply:
Fixed.